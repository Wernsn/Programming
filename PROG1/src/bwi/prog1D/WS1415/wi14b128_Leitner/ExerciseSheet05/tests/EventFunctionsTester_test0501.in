5
title
2000
1
1
15
tItle
2050
1
1
5
1
5
tItle
2000
1
1
15
title
2050
1
1
5
1
5
title
2000
1
1
15
title
2050
1
1
5
1
5
title
2000
1
1
15
tItle
2050
1
1
5
2
5
title
2050
1
1
15
tItle
2040
1
1
5
2
5
title
2000
1
1
15
tItle
2000
1
1
5
2

5
title
2000
1
1
15
tItle
2050
1
1
5
3
5
title
2000
1
1
30
tItle
2050
1
1
5
3
5
title
2000
1
1
3
tItle
2050
1
1
3
3
0
