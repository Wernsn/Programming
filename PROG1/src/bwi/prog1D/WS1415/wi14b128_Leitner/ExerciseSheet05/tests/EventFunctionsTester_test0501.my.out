compare: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: 1
compare: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: -1
compare: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: 0
compare: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: -1
compare: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: 1
compare: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: 0
compare: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: 1
compare: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: 1
compare: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: 0
